edb/server/compiler/sertypes.py

```diff
@@ -480,7 +481,9 @@ def parse(cls, typedesc: bytes) -> TypeDesc:
             desc = cls._parse(wrapped, codecs_list)
             if desc is not None:
                 codecs_list.append(desc)
-        return desc
+        if not codecs_list:
+            raise errors.InternalServerError('could not parse type descriptor')
+        return codecs_list[-1]
```

Comment:
Not sure how this is related to the issue...

Reply:
It's in a separate commit